src/instructlab/model/download.py

```diff
@@ -70,7 +72,7 @@ def download(self):
         """
         Download specified model from Hugging Face
         """
-        click.echo(
+        logger.info(
             f"Downloading model from Hugging Face:\n{DEFAULT_INDENT}Model: {self.repository}@{self.release}\n{DEFAULT_INDENT}Destination: {self.download_dest}"
         )
```

Comment:
So we will use `logger.info` instead of `click.echo` in the future? Seems will different if in log.

Reply:
@reidliu41 we are restricting click to the `src/instructlab/cli` package. That package serves as the "client" side of the architecture so click should only be used there
This file represents the logical "backend" so it should not use click anywhere

Reply:
thanks for share.